KnowWhereGraph/node-browser
# Node Explorer 1.0.0

# Node Explorer 1.0.0

First major release of the node explorer. 

Highlights
- Angular based
- Multi-environment deployment strategies
- Visually _similar_ to phuzzy.link
- Some existing phuzzy.link bugs fixed (eg not all relations being shown)

## Testing
1. Clone the repo
2. npm i
3. ng serve --configuration=local
4. Visit http://localhost:4200/browse#kwgr:NWZone.30067 to see a map
5. Visit http://localhost:4200/browse#kwgr:hazard.1183610.5434008 to see a node without a map

<img width="1728" alt="Screenshot 2023-09-30 at 12 34 13 PM" src="https://github.com/KnowWhereGraph/node-browser/assets/9289652/1f17ac65-5678-4c3c-9d3b-bc6d21d28fe2">
![Uploading Screen
<img width="1728" alt="Screenshot 2023-09-30 at 12 54 22 PM" src="https://github.com/KnowWhereGraph/node-browser/assets/9289652/adac9767-572e-4c08-901a-511dc64a64b7">
